AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -937,14 +935,12 @@ function Request-AzPowerShellModule {
     Remove-Module -Name Az.Accounts -Force -ErrorAction SilentlyContinue
     Remove-Module -Name Az.Network -Force -ErrorAction SilentlyContinue
 
-    $storageModule = ,(Get-Module -Name Az.Storage -ListAvailable | `

```

Comment:
Is there a reason this was removed? The reason I did this is because if the customer ends up with two modules instead we want to import the newer one. PowerShell has this sometimes useful, sometimes annoying feature where it auto-unrolls collections of one. The comma prevents that behavior...

Reply:
I removed this because I thought we only support one version now. But just realized that we should allow versions higher than 2.0.0, so now I am adding back the comma operator.